test/functional/cmdLineTests/SystemPropertiesTest/src/SysPropTest.java

```diff
@@ -57,7 +57,7 @@ public static void main(String args[])
 				if ((argEncoding.equals("UTF-8") || argEncoding.equals("ISO-8859-1"))) {
 					strTestProp = new String(B,argEncoding);
 				} else {
-					strTestProp = new String(B);
+					strTestProp = new String(B, System.getProperty("native.encoding"));
```

Comment:
native.encoding is only available from jdk17, is this change going to work on prior versions?

Reply:
Grinder on jdk8 fails with NPE. 
https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/618

Reply:
Use `Charset.defaultCharset().name()` as default value.